plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -266,11 +266,8 @@ private IcebergFileWriter createOrcWriter(
     public static OrcWriterOptions withBloomFilterOptions(OrcWriterOptions orcWriterOptions, Map<String, String> storageProperties)
     {
         if (storageProperties.containsKey(ORC_BLOOM_FILTER_COLUMNS_KEY)) {
-            if (!storageProperties.containsKey(ORC_BLOOM_FILTER_FPP_KEY)) {
-                throw new TrinoException(ICEBERG_INVALID_METADATA, "FPP for Bloom filter is missing");
-            }
             try {
-                double fpp = parseDouble(storageProperties.get(ORC_BLOOM_FILTER_FPP_KEY));
+                double fpp = storageProperties.containsKey(ORC_BLOOM_FILTER_FPP_KEY) ? parseDouble(storageProperties.get(ORC_BLOOM_FILTER_FPP_KEY)) : orcWriterOptions.getBloomFilterFpp();
```

Comment:
nit: Can we split them into two lines ?

Reply:
splitted to three lines for better readability :)